aws_lambda_powertools/utilities/parser/models/apigw.py

```diff
@@ -1,11 +1,10 @@
+# ruff: noqa: FA100
```

Comment:
We don't need to add this line in every file, we can make it global in the `ruff.toml` file.

```
[lint.per-file-ignores]
# Ignore specific rules for specific files
"tests/e2e/utils/data_builder/__init__.py" = ["F401"]
"tests/e2e/utils/data_fetcher/__init__.py" = ["F401"]
"aws_lambda_powertools/utilities/data_classes/s3_event.py" = ["A003"]
"aws_lambda_powertools/utilities/parser/models/__init__.py" = ["E402"]
"aws_lambda_powertools/event_handler/openapi/compat.py" = ["F401"]
# Maintenance: we're keeping EphemeralMetrics code in case of Hyrum's law so we can quickly revert it
"aws_lambda_powertools/metrics/metrics.py" = ["ERA001"]
"aws_lambda_powertools/utilities/parser/models/*" = ["FA100"] <--- Add this line
```

Reply:
Done